src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -472,6 +472,8 @@ public enum SslProtocols
         Tls = 192,
         Tls11 = 768,
         Tls12 = 3072,
+        [Obsolete("This value has been deprecated.  It is no longer supported. http://go.microsoft.com/fwlink/?linkid=14202")]
```

Comment:
/cc @ericstj 

@danmosemsft Please don't add this to the implementation as it may break certain scenarios using Xml serializer.
The only reason this is not yet deprecated in NetFX (together with Ssl2,3) is that the XML seralizer/deserializer will break/throw if it tries to deserialize an Obsolete attribute.